package.json

```diff
@@ -16,7 +16,7 @@
     "postinstall": "husky install",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
-    "test:integration": "jest --setupFiles ./.integration.test-${ENV:-build}.env -c jest.integration.config.ts"
+    "test:integration": "jest -c jest.integration.config.ts -i"
```

Comment:
Please can we not add an `-i` flag. Tests should be designed to run in parallel.

Reply:
Done.